.github/workflows/labeler.yml

```diff
@@ -0,0 +1,63 @@
+name: "Pull Request Labeler"
+
+on:
+  pull_request_target:
+    types: ["opened"]
+    branches: ["master", "main"]
+
+jobs:
+  labeler:
+    if: github.event.pull_request.user.login == 'devex-sa'
```

Comment:
I think this should be read from a org secret to avoid exposing too much information about our environment and users.

Reply:
It's unfortunately defined here:

https://github.com/dfds/infrastructure-modules/blob/a5893dbf673c3e6d97efefff4354ab1da7fd0c2d/renovate.json#L4

And we can't use secrets in a json.